README.md

```diff
@@ -1,25 +1,74 @@
-# Code Guidelines
-For reference, see the below guidelines we use for peer code review at Exodus.
+# Code and Process Guidelines
+
+For reference, see the below guidelines we use for peer code review at Exodus. You can follow this checklist when evaluating a code challenge submission.
+
+## Grading
+
+See points in parentheses before each item. Negative points means the candidate can only lose points for that item. The expected Total Score:
+
+- Candidates should score a total of >15 points if they use PRs, >10 otherwise
+- Colleagues should score a total of >20 points
+
+## Code
+
+> [!IMPORTANT]
+> GRADING (SECTION SCORE)
+>
+> - Everyone should score at least 10 points (tests optional for candidates)
+
+### Functionality
+
+- [ ] (1) All task requirements are fully implemented, tradeoffs made are documented, explanations are provided for any missing pieces
+- [ ] (1) Not obviously broken
+- [ ] (1) No dubious functionality on top of the spec
+- [ ] (1) Not over-engineered, e.g., doesn’t prematurely optimize at the expense of complexity
+
+### Code Quality
+
+- [ ] (2) Code is readable: linted, prettified, and written in a consistent style
+- [ ] (1) Code is self-documenting: semantic variables names, function names describe implemented function
+- [ ] (1) Code is as simple as possible and modular
+  - Small single-purpose functions > giant functions that do 10 things
+  - Code is logically separated (e.g., no giant utils folders)
+- [ ] (2) Author uses objects, arrays, inheritance, closures, callbacks, promises expertly, and is up to date with the language/technology, e.g., async/- [ ] await > promise chains > callbacks for code after 2019
+- [ ] (1) Tests are included and:
+  - Are readable (intent is clear)
+  - Cover happy path
+  - Cover edge cases
+
+### Documentation
+
+- [ ] (2) Clear instructions for project setup
+- [ ] (-1) Clear code comments, but _only when necessary_. Comments are for the why, not the what
```

Comment:
What if we start a "Negatives" section and breakout all of the offensive code smells there?

Reply:
we could, it just depends on how short we want to keep this. Or we can have a separate phase to the challenge, e.g. please stop after 7 hours and spend the last hour working on refactoring towards our coding best practices

Reply:
Let's keep it like you have it then.  I always assume that most ppl spend more than 8 hours so any time refactoring will be spent by someone who is determined to do well anyway